ucac/migrations/2_deploy_contracts.js

```diff
@@ -150,5 +190,37 @@ module.exports = async function (deployer, network, accounts) {
                                                        , sek
                                                        , web3.toWei(100)
                                                        , {from: web3.eth.accounts[0]});
+        await creditProtocolContract.createAndStakeUcac( lndrIdrContract.address
+                                                        , idr
+                                                        , web3.toWei(100)
+                                                        , {from: web3.eth.accounts[0]});
+        await creditProtocolContract.createAndStakeUcac( lndrMyrContract.address
+                                                        , myr
+                                                        , web3.toWei(100)
+                                                        , {from: web3.eth.accounts[0]});
+        await creditProtocolContract.createAndStakeUcac( lndrSgdContract.address
+                                                        , sgd
+                                                        , web3.toWei(100)
+                                                        , {from: web3.eth.accounts[0]});
+        await creditProtocolContract.createAndStakeUcac( lndrThbContract.address
+                                                        , thb
+                                                        , web3.toWei(100)
+                                                        , {from: web3.eth.accounts[0]});
+        await creditProtocolContract.createAndStakeUcac( lndrVndContract.address
+                                                        , vnd
+                                                        , web3.toWei(100)
+                                                        , {from: web3.eth.accounts[0]});
+        await creditProtocolContract.createAndStakeUcac( lndrIlsContract.address
+                                                        , ils
+                                                        , web3.toWei(100)
+                                                        , {from: web3.eth.accounts[0]});
+        await creditProtocolContract.createAndStakeUcac( lndrRubContract.address
+                                                        , rub
+                                                        , web3.toWei(100)
+                                                        , {from: web3.eth.accounts[0]});
+        await creditProtocolContract.createAndStakeUcac( lndrTryContract.address
+                                                        , Try
```

Comment:
🤔Ha! I thought this was a typo at first, but I see it's just a workaround for the reserved word collision.